tests/integration_tests/lightgbm_tuner_tests/test_optimize.py

```diff
@@ -1,8 +1,13 @@
 import contextlib
-
 import mock
 import numpy as np
 import pytest
+from typing import Any
+from typing import Dict
+from typing import Generator
+from typing import List
+from typing import Optional
+import warnings
```

Comment:
Ditto regarding import order.

Reply:
Thanks! I fixed it in https://github.com/optuna/optuna/pull/1032/commits/7f0adbf47d26ba28422cd2e5e41024473d67efe7.